check_pve.py

```diff
@@ -186,7 +191,7 @@ def check_vm_status(self, idx, **kwargs):
                 break
 
         if not found:
-            self.check_message = "VM '{}' not found".format(idx)
+            self.check_message = "VM or LXC named '{}' not found".format(idx)
```

Comment:
I would suggest to remove the `named` as the identifier could be the `name` or the `vmid`

Reply:
You're right, thanks for pointing that out. Hopefully fixed in 75aebf8 :)